pkg/repository/workspace_redis.go

```diff
@@ -16,6 +18,8 @@ func NewWorkspaceRedisRepository(r *common.RedisClient) WorkspaceRepository {
 	return &WorkspaceRedisRepository{rdb: r}
 }
 
+var CACHED_TOKEN_TTL_S = 600 // 10 minutes
```

Comment:
Another nitpick, can this be `cachedTokenTTlS` instead? In addition can it be a `const`?

Reply:
oh yea. I think i mixed up some conventions